tsl/generateWarpScript.go

```diff
@@ -1319,6 +1327,47 @@ func (protoParser *ProtoParser) removeLabels(framework FrameworkStatement) (stri
 	return value, nil
 }
 
+// operators generate WarpScript line for an individual statement
+func (protoParser *ProtoParser) renameTemplate(framework FrameworkStatement) (string, error) {
+	value := ""
+
+	if attribute, ok := framework.unNamedAttributes[0]; ok {
+		if attribute.tokenType != STRING {
+			message := "Rename template expects a its parameter to be a STRING"
+			return "", protoParser.NewProtoError(message, framework.pos)
+		}
+
+		value = "<% DROP DUP 'series' STORE [ '" + attribute.lit + "' ]"
+		if strings.Contains(value, "${this.name}") {
+			value = strings.Replace(value, "${this.name}", "' $series NAME '", -1)
```

Comment:
is `${ name }` working ?

Reply:
${name} can't work as it will replace ${name} by the content of the TSL variable called name. That's why we have to prefix any call to the series object by this.

Reply:
But now {{ this.name }} is working ;)